### PR TITLE
DG - Add enhancements, update NFRs, add line breaks

### PR DESCRIPTION
**Pull request by UdhayaShan1**
- Add NFRs
- Add planned enhancements
- Add appendix effort
- Add appendix manual testing

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -1196,20 +1340,86 @@ testers are expected to do more *exploratory* testing.
...
+6. **Make `PHONE` input conditions less restrictive**
+   - Currently, `PHONE` only allows numbers, and it should be at least three digits long
+   - This poses problems to users who wish to input 
+     - Symbols such `+6590215365` or `1-800` numbers
+       which may commonly have `-`.
+     - Symbols such as white-space as they may wish to put gaps such as `9021 5365`
+   - We plan to reduce the restrictions of no symbols for future iterations 
+
+7. **Allow symbols in `NAME` input**
+   - Currently, `NAME` does not accept symbols, hence common substrings such as `s/o` or `d/o` are not allowed in the name input.
```
Comment (4llysa) [inline docs/DeveloperGuide.md:1406]:
```suggestion
   - Currently, `NAME` does not accept symbols, hence common substrings such as `s/o`, `d/o` or punctuation such as "Tan Xin En, Betty" are not allowed in the name input.
```

Comment (UdhayaShan1) [reply docs/DeveloperGuide.md]: ty

**Comment by 4llysa** [comment]
for 8,9,10 i'm thinking about

- No support for multiple fields. For E.g., A seller may be selling multiple properties and you want to list multiple addresses. A client may have multiple phone numbers for mobile / office phones but may only have one. 
- Missing important fields like district number that would be extremely useful for real estate agents
-  `help clear` should be `help clearRealodex` for consistency

**Comment by UdhayaShan1** [comment]
I think i will add family and income symbols for 9 and 10 it might be more FFish

**Review by wapisai — APPROVED** [review]
lgtm

**Pull request merged**